mongodb/common/.classpath

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
```

Comment:
Let's not add .classpath or other IDE-specific files to the source control. Please just add a .gitignore entry for it.